packages/webapp/src/views/developer/details-email.html

```diff
@@ -0,0 +1,3 @@
+{% set serviceName = "Record biodiversity gains" %}
+
+{% include 'land/email.html' %}
```

Comment:
I think you're going to have the same problem with the email as the name view, needs to be its own view or both landowner and developer journey views need to pull in a partial